pip.ado

```diff
@@ -476,7 +464,7 @@ qui {
 		noi pip_cl, country("`country'")  /// this needs to accommodate to new structure
 		year("`year'")                    ///
 		povline("`povline'")              ///
-		ppp("`ppp_year'")                      ///
+		ppp_year("`ppp_year'")            ///
```

Comment:
the `pip_cl` function is not in use right now, but if you change one of the paramters, you must make sure it is sure in the original pip_cl.ado. Thanks.